src/components/Search/Search.js

```diff
@@ -1,47 +1,146 @@
-import React from 'react';
+import React, { createRef } from 'react';
+import { Link } from 'react-router-dom';
 import './Search.scss';
+import SearchList from '../../components/SearchList/SearchList';
```

Comment:
- Import 순서 수정해주세요! 일반적인 convention을 따르는 이유도 있지만 순서만 잘 지켜주셔도 가독성이 좋아집니다. 아래 순서 참고해주세요.
- React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)
```suggestion
import React, { createRef } from 'react';
import { Link } from 'react-router-dom';
import SearchList from '../../components/SearchList/SearchList';
import './Search.scss';
```

Reply:
수정 완료 되었습니다😎